django-stubs/contrib/postgres/expressions.pyi

```diff
@@ -0,0 +1,9 @@
+from django.contrib.postgres.fields import ArrayField
+from django.db.models import Subquery
+from django.utils.functional import cached_property
+
+class ArraySubquery(Subquery):
+    template: str
+
+    @cached_property
+    def output_field(self) -> ArrayField: ...
```

Comment:
Need allowlist entry for `django.contrib.postgres.expressions.ArraySubquery.output_field` due to `@cached_property`.